2018-07-07 00:07:54.390 [error] <0.279.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.284.0> exit with reason normal in context child_terminated
2018-07-07 00:07:54.434 [info] <0.33.0> Application lager started on node nonode@nohost
2018-07-07 00:07:54.445 [info] <0.33.0> Application xmerl started on node nonode@nohost
2018-07-07 00:07:54.449 [info] <0.33.0> Application jsx started on node nonode@nohost
2018-07-07 00:07:54.451 [info] <0.33.0> Application ranch_proxy_protocol started on node nonode@nohost
2018-07-07 00:07:54.454 [info] <0.33.0> Application recon started on node nonode@nohost
2018-07-07 00:07:54.454 [info] <0.33.0> Application rabbit_common started on node nonode@nohost
2018-07-07 00:07:54.464 [info] <0.33.0> Application amqp_client started on node nonode@nohost
2018-07-07 00:07:54.464 [info] <0.33.0> Application amqp started on node nonode@nohost
2018-07-07 00:07:54.466 [info] <0.33.0> Application agala started on node nonode@nohost
2018-07-07 00:07:54.466 [info] <0.33.0> Application agala_fb started on node nonode@nohost
2018-07-07 00:07:55.114 [warning] <0.32.0> lager_error_logger_h dropped 69 messages in the last second that exceeded the limit of 50 messages/sec
2018-07-07 00:08:16.730 [error] <0.464.0> CRASH REPORT Process <0.464.0> with 0 neighbours exited with reason: {{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,exports => nil,function => call,module => 'Elixir.Awe.Chains.Fb',reason => nil},[{'Elixir.Awe.Chains.Fb',call,[#{'__struct__' => 'Elixir.Agala.Conn',assigns => #{},fallback => nil,halted => false,multi => nil,private => #{},request => <<"{\"entry\": [{\"changes\": [{\"field\": \"messages\", \"value\": {\"page_id\": \"1067280970047460\"}}], \"id\": \"0\", \"time\": 1530911292}], \"object\": \"page\"}">>,...}],...},...]},...} in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20 in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20
2018-07-07 00:08:16.780 [error] <0.331.0> Lager event handler error_logger_lager_h exited with reason {'EXIT',{{case_clause,['Elixir.Awe.Router.HTTP',<0.463.0>,1,<0.464.0>,{{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,exports => nil,function => call,module => 'Elixir.Awe.Chains.Fb',reason => nil},[{'Elixir.Awe.Chains.Fb',call,[#{'__struct__' => 'Elixir.Agala.Conn',assigns => #{},fallback => nil,halted => false,multi => nil,private => #{},request => <<"{\"entry\": [{\"changes\": [{\"field\": \"messages\", \"value\": {\"page_id\": \"1067280970047460\"}}], \"id\": \"0...">>,...}],...},...]},...},...]},...}}
2018-07-07 00:08:52.893 [error] <0.468.0> CRASH REPORT Process <0.468.0> with 0 neighbours exited with reason: {{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,exports => nil,function => call,module => 'Elixir.Awe.Chains.Fb',reason => nil},[{'Elixir.Awe.Chains.Fb',call,[#{'__struct__' => 'Elixir.Agala.Conn',assigns => #{},fallback => nil,halted => false,multi => nil,private => #{},request => <<"{\"entry\": [{\"changes\": [{\"field\": \"messages\", \"value\": {\"page_id\": \"1067280970047460\"}}], \"id\": \"0\", \"time\": 1530911292}], \"object\": \"page\"}">>,...}],...},...]},...} in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20 in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20
2018-07-07 00:08:52.896 [error] <0.331.0> Lager event handler error_logger_lager_h exited with reason {'EXIT',{{case_clause,['Elixir.Awe.Router.HTTP',<0.467.0>,1,<0.468.0>,{{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,exports => nil,function => call,module => 'Elixir.Awe.Chains.Fb',reason => nil},[{'Elixir.Awe.Chains.Fb',call,[#{'__struct__' => 'Elixir.Agala.Conn',assigns => #{},fallback => nil,halted => false,multi => nil,private => #{},request => <<"{\"entry\": [{\"changes\": [{\"field\": \"messages\", \"value\": {\"page_id\": \"1067280970047460\"}}], \"id\": \"0...">>,...}],...},...]},...},...]},...}}
2018-07-07 00:08:53.610 [error] <0.471.0> CRASH REPORT Process <0.471.0> with 0 neighbours exited with reason: {{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,exports => nil,function => call,module => 'Elixir.Awe.Chains.Fb',reason => nil},[{'Elixir.Awe.Chains.Fb',call,[#{'__struct__' => 'Elixir.Agala.Conn',assigns => #{},fallback => nil,halted => false,multi => nil,private => #{},request => <<"{\"object\":\"page\",\"entry\":[{\"id\":\"730671640616281\",\"time\":1530911333352,\"messaging\":[{\"sender\":{\"id\":\"1468676106567600\"},\"recipient\":{\"id\":\"730671640616281\"},\"timestamp\":...">>,...}],...},...]},...} in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20 in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20
2018-07-07 00:08:53.611 [error] <0.331.0> Lager event handler error_logger_lager_h exited with reason {'EXIT',{{case_clause,['Elixir.Awe.Router.HTTP',<0.470.0>,1,<0.471.0>,{{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,exports => nil,function => call,module => 'Elixir.Awe.Chains.Fb',reason => nil},[{'Elixir.Awe.Chains.Fb',call,[#{'__struct__' => 'Elixir.Agala.Conn',assigns => #{},fallback => nil,halted => false,multi => nil,private => #{},request => <<"{\"object\":\"page\",\"entry\":[{\"id\":\"730671640616281\",\"time\":1530911333352,\"messaging\":[{\"sender\":{\"id...">>,...}],...},...]},...},...]},...}}
2018-07-07 00:09:45.294 [error] <0.268.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.273.0> exit with reason normal in context child_terminated
2018-07-07 00:09:45.336 [info] <0.33.0> Application lager started on node nonode@nohost
2018-07-07 00:09:45.342 [info] <0.33.0> Application xmerl started on node nonode@nohost
2018-07-07 00:09:45.346 [info] <0.33.0> Application jsx started on node nonode@nohost
2018-07-07 00:09:45.348 [info] <0.33.0> Application ranch_proxy_protocol started on node nonode@nohost
2018-07-07 00:09:45.350 [info] <0.33.0> Application recon started on node nonode@nohost
2018-07-07 00:09:45.350 [info] <0.33.0> Application rabbit_common started on node nonode@nohost
2018-07-07 00:09:45.357 [info] <0.33.0> Application amqp_client started on node nonode@nohost
2018-07-07 00:09:45.357 [info] <0.33.0> Application amqp started on node nonode@nohost
2018-07-07 00:09:45.360 [info] <0.33.0> Application agala started on node nonode@nohost
2018-07-07 00:09:45.360 [info] <0.33.0> Application agala_fb started on node nonode@nohost
2018-07-07 00:09:46.106 [warning] <0.32.0> lager_error_logger_h dropped 69 messages in the last second that exceeded the limit of 50 messages/sec
2018-07-07 00:09:58.388 [error] <0.453.0> CRASH REPORT Process <0.453.0> with 0 neighbours exited with reason: {{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 2,exports => nil,function => response,module => 'Elixir.Agala.Provider.Facebook.Controllers.View',reason => nil},[{'Elixir.Agala.Provider.Facebook.Controllers.View',response,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Adapters.Cowboy2.Conn',#{bindings => #{},body_length => 140,cert => undefined,has_body => true,has_read_body => true,headers => #{<<"accept">> => <<"*/*">>,<<"connect...">> => ...,...},...}},...},...],...},...]},...} in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20 in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20
2018-07-07 00:09:58.436 [error] <0.320.0> Lager event handler error_logger_lager_h exited with reason {'EXIT',{{case_clause,['Elixir.Awe.Router.HTTP',<0.452.0>,1,<0.453.0>,{{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 2,exports => nil,function => response,module => 'Elixir.Agala.Provider.Facebook.Controllers.View',reason => nil},[{'Elixir.Agala.Provider.Facebook.Controllers.View',response,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Adapters.Cowboy2.Conn',#{bindings => #{},body_length => 140,cert => undefined,has_body => true,has_read_body => ...,...}},...},...],...},...]},...},...]},...}}
2018-07-07 00:10:05.962 [error] <0.458.0> CRASH REPORT Process <0.458.0> with 0 neighbours exited with reason: {{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 2,exports => nil,function => response,module => 'Elixir.Agala.Provider.Facebook.Controllers.View',reason => nil},[{'Elixir.Agala.Provider.Facebook.Controllers.View',response,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Adapters.Cowboy2.Conn',#{bindings => #{},body_length => 140,cert => undefined,has_body => true,has_read_body => true,headers => #{<<"accept">> => <<"*/*">>,<<"connect...">> => ...,...},...}},...},...],...},...]},...} in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20 in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20
2018-07-07 00:10:05.963 [error] <0.320.0> Lager event handler error_logger_lager_h exited with reason {'EXIT',{{case_clause,['Elixir.Awe.Router.HTTP',<0.457.0>,1,<0.458.0>,{{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 2,exports => nil,function => response,module => 'Elixir.Agala.Provider.Facebook.Controllers.View',reason => nil},[{'Elixir.Agala.Provider.Facebook.Controllers.View',response,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Adapters.Cowboy2.Conn',#{bindings => #{},body_length => 140,cert => undefined,has_body => true,has_read_body => ...,...}},...},...],...},...]},...},...]},...}}
2018-07-07 00:10:13.072 [error] <0.461.0> CRASH REPORT Process <0.461.0> with 0 neighbours exited with reason: {{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 2,exports => nil,function => response,module => 'Elixir.Agala.Provider.Facebook.Controllers.View',reason => nil},[{'Elixir.Agala.Provider.Facebook.Controllers.View',response,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Adapters.Cowboy2.Conn',#{bindings => #{},body_length => 321,cert => undefined,has_body => true,has_read_body => true,headers => #{<<"accept">> => <<"*/*">>,<<"accept-...">> => ...,...},...}},...},...],...},...]},...} in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20 in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20
2018-07-07 00:10:13.073 [error] <0.320.0> Lager event handler error_logger_lager_h exited with reason {'EXIT',{{case_clause,['Elixir.Awe.Router.HTTP',<0.460.0>,1,<0.461.0>,{{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 2,exports => nil,function => response,module => 'Elixir.Agala.Provider.Facebook.Controllers.View',reason => nil},[{'Elixir.Agala.Provider.Facebook.Controllers.View',response,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Adapters.Cowboy2.Conn',#{bindings => #{},body_length => 321,cert => undefined,has_body => true,has_read_body => ...,...}},...},...],...},...]},...},...]},...}}
2018-07-07 00:10:40.751 [error] <0.464.0> CRASH REPORT Process <0.464.0> with 0 neighbours exited with reason: {{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 2,exports => nil,function => response,module => 'Elixir.Agala.Provider.Facebook.Controllers.View',reason => nil},[{'Elixir.Agala.Provider.Facebook.Controllers.View',response,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Adapters.Cowboy2.Conn',#{bindings => #{},body_length => 140,cert => undefined,has_body => true,has_read_body => true,headers => #{<<"accept">> => <<"*/*">>,<<"connect...">> => ...,...},...}},...},...],...},...]},...} in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20 in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20
2018-07-07 00:10:40.752 [error] <0.320.0> Lager event handler error_logger_lager_h exited with reason {'EXIT',{{case_clause,['Elixir.Awe.Router.HTTP',<0.463.0>,1,<0.464.0>,{{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 2,exports => nil,function => response,module => 'Elixir.Agala.Provider.Facebook.Controllers.View',reason => nil},[{'Elixir.Agala.Provider.Facebook.Controllers.View',response,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Adapters.Cowboy2.Conn',#{bindings => #{},body_length => 140,cert => undefined,has_body => true,has_read_body => ...,...}},...},...],...},...]},...},...]},...}}
2018-07-07 00:10:48.946 [error] <0.259.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.264.0> exit with reason normal in context child_terminated
2018-07-07 00:10:48.979 [info] <0.33.0> Application lager started on node nonode@nohost
2018-07-07 00:10:48.984 [info] <0.33.0> Application xmerl started on node nonode@nohost
2018-07-07 00:10:48.988 [info] <0.33.0> Application jsx started on node nonode@nohost
2018-07-07 00:10:48.990 [info] <0.33.0> Application ranch_proxy_protocol started on node nonode@nohost
2018-07-07 00:10:48.992 [info] <0.33.0> Application recon started on node nonode@nohost
2018-07-07 00:10:48.992 [info] <0.33.0> Application rabbit_common started on node nonode@nohost
2018-07-07 00:10:48.999 [info] <0.33.0> Application amqp_client started on node nonode@nohost
2018-07-07 00:10:48.999 [info] <0.33.0> Application amqp started on node nonode@nohost
2018-07-07 00:10:49.001 [info] <0.33.0> Application agala started on node nonode@nohost
2018-07-07 00:10:49.001 [info] <0.33.0> Application agala_fb started on node nonode@nohost
2018-07-07 00:10:50.000 [warning] <0.32.0> lager_error_logger_h dropped 19 messages in the last second that exceeded the limit of 50 messages/sec
2018-07-07 00:12:03.541 [error] <0.251.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.256.0> exit with reason normal in context child_terminated
2018-07-07 00:12:03.588 [info] <0.33.0> Application lager started on node nonode@nohost
2018-07-07 00:12:03.594 [info] <0.33.0> Application xmerl started on node nonode@nohost
2018-07-07 00:12:03.597 [info] <0.33.0> Application jsx started on node nonode@nohost
2018-07-07 00:12:03.600 [info] <0.33.0> Application ranch_proxy_protocol started on node nonode@nohost
2018-07-07 00:12:03.602 [info] <0.33.0> Application recon started on node nonode@nohost
2018-07-07 00:12:03.602 [info] <0.33.0> Application rabbit_common started on node nonode@nohost
2018-07-07 00:12:03.607 [info] <0.33.0> Application amqp_client started on node nonode@nohost
2018-07-07 00:12:03.607 [info] <0.33.0> Application amqp started on node nonode@nohost
2018-07-07 00:12:03.608 [info] <0.33.0> Application agala started on node nonode@nohost
2018-07-07 00:12:03.608 [info] <0.33.0> Application agala_fb started on node nonode@nohost
2018-07-07 00:12:04.100 [warning] <0.32.0> lager_error_logger_h dropped 61 messages in the last second that exceeded the limit of 50 messages/sec
2018-07-09 23:43:20.717 [error] <0.410.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.415.0> exit with reason normal in context child_terminated
2018-07-09 23:43:20.766 [info] <0.33.0> Application lager started on node nonode@nohost
2018-07-09 23:43:20.773 [info] <0.33.0> Application xmerl started on node nonode@nohost
2018-07-09 23:43:20.778 [info] <0.33.0> Application jsx started on node nonode@nohost
2018-07-09 23:43:20.780 [info] <0.33.0> Application ranch_proxy_protocol started on node nonode@nohost
2018-07-09 23:43:20.782 [info] <0.33.0> Application recon started on node nonode@nohost
2018-07-09 23:43:20.782 [info] <0.33.0> Application rabbit_common started on node nonode@nohost
2018-07-09 23:43:20.791 [info] <0.33.0> Application amqp_client started on node nonode@nohost
2018-07-09 23:43:20.791 [info] <0.33.0> Application amqp started on node nonode@nohost
2018-07-09 23:43:20.793 [info] <0.33.0> Application agala started on node nonode@nohost
2018-07-09 23:43:20.793 [info] <0.33.0> Application agala_fb started on node nonode@nohost
2018-07-09 23:43:21.110 [warning] <0.32.0> lager_error_logger_h dropped 69 messages in the last second that exceeded the limit of 50 messages/sec
2018-07-09 23:49:19.038 [error] <0.310.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.315.0> exit with reason normal in context child_terminated
2018-07-09 23:49:19.089 [info] <0.33.0> Application lager started on node nonode@nohost
2018-07-09 23:49:19.099 [info] <0.33.0> Application xmerl started on node nonode@nohost
2018-07-09 23:49:19.103 [info] <0.33.0> Application jsx started on node nonode@nohost
2018-07-09 23:49:19.105 [info] <0.33.0> Application ranch_proxy_protocol started on node nonode@nohost
2018-07-09 23:49:19.107 [info] <0.33.0> Application recon started on node nonode@nohost
2018-07-09 23:49:19.107 [info] <0.33.0> Application rabbit_common started on node nonode@nohost
2018-07-09 23:49:19.116 [info] <0.33.0> Application amqp_client started on node nonode@nohost
2018-07-09 23:49:19.117 [info] <0.33.0> Application amqp started on node nonode@nohost
2018-07-09 23:49:19.120 [info] <0.33.0> Application agala started on node nonode@nohost
2018-07-09 23:49:19.120 [info] <0.33.0> Application agala_fb started on node nonode@nohost
2018-07-09 23:49:20.127 [warning] <0.32.0> lager_error_logger_h dropped 69 messages in the last second that exceeded the limit of 50 messages/sec
2018-07-09 23:50:30.709 [error] <0.310.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.315.0> exit with reason normal in context child_terminated
2018-07-09 23:50:30.756 [info] <0.33.0> Application lager started on node nonode@nohost
2018-07-09 23:50:30.766 [info] <0.33.0> Application xmerl started on node nonode@nohost
2018-07-09 23:50:30.770 [info] <0.33.0> Application jsx started on node nonode@nohost
2018-07-09 23:50:30.775 [info] <0.33.0> Application ranch_proxy_protocol started on node nonode@nohost
2018-07-09 23:50:30.778 [info] <0.33.0> Application recon started on node nonode@nohost
2018-07-09 23:50:30.778 [info] <0.33.0> Application rabbit_common started on node nonode@nohost
2018-07-09 23:50:30.786 [info] <0.33.0> Application amqp_client started on node nonode@nohost
2018-07-09 23:50:30.786 [info] <0.33.0> Application amqp started on node nonode@nohost
2018-07-09 23:50:30.789 [info] <0.33.0> Application agala started on node nonode@nohost
2018-07-09 23:50:30.789 [info] <0.33.0> Application agala_fb started on node nonode@nohost
2018-07-09 23:50:31.130 [warning] <0.32.0> lager_error_logger_h dropped 69 messages in the last second that exceeded the limit of 50 messages/sec
2018-07-09 23:50:41.959 [error] <0.508.0> CRASH REPORT Process <0.508.0> with 0 neighbours exited with reason: {{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,exports => nil,function => call,module => 'Elixir.Awe.Chains.Fb',reason => nil},[{'Elixir.Awe.Chains.Fb',call,[#{'__struct__' => 'Elixir.Agala.Conn',assigns => #{},fallback => nil,halted => false,multi => nil,private => #{},request => <<"{\"entry\": [{\"changes\": [{\"field\": \"messages\", \"value\": {\"page_id\": \"1067280970047460\"}}], \"id\": \"0\", \"time\": 1531169438}], \"object\": \"page\"}">>,...}],...},...]},...} in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20 in 'Elixir.Plug.Adapters.Cowboy2.Handler':init/2 line 20
2018-07-09 23:50:42.007 [error] <0.373.0> Lager event handler error_logger_lager_h exited with reason {'EXIT',{{case_clause,['Elixir.Awe.Router.HTTP',<0.507.0>,1,<0.508.0>,{{#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,exports => nil,function => call,module => 'Elixir.Awe.Chains.Fb',reason => nil},[{'Elixir.Awe.Chains.Fb',call,[#{'__struct__' => 'Elixir.Agala.Conn',assigns => #{},fallback => nil,halted => false,multi => nil,private => #{},request => <<"{\"entry\": [{\"changes\": [{\"field\": \"messages\", \"value\": {\"page_id\": \"1067280970047460\"}}], \"id\": \"0...">>,...}],...},...]},...},...]},...}}
2018-07-09 23:50:56.034 [error] <0.310.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.315.0> exit with reason normal in context child_terminated
2018-07-09 23:50:56.078 [info] <0.33.0> Application lager started on node nonode@nohost
2018-07-09 23:50:56.086 [info] <0.33.0> Application xmerl started on node nonode@nohost
2018-07-09 23:50:56.092 [info] <0.33.0> Application jsx started on node nonode@nohost
2018-07-09 23:50:56.095 [info] <0.33.0> Application ranch_proxy_protocol started on node nonode@nohost
2018-07-09 23:50:56.099 [info] <0.33.0> Application recon started on node nonode@nohost
2018-07-09 23:50:56.099 [info] <0.33.0> Application rabbit_common started on node nonode@nohost
2018-07-09 23:50:56.106 [info] <0.33.0> Application amqp_client started on node nonode@nohost
2018-07-09 23:50:56.106 [info] <0.33.0> Application amqp started on node nonode@nohost
2018-07-09 23:50:56.109 [info] <0.33.0> Application agala started on node nonode@nohost
2018-07-09 23:50:56.109 [info] <0.33.0> Application agala_fb started on node nonode@nohost
2018-07-09 23:50:57.132 [warning] <0.32.0> lager_error_logger_h dropped 69 messages in the last second that exceeded the limit of 50 messages/sec
